doc/contents.md

```diff
@@ -0,0 +1,31 @@
+DLite documentation
+===================
+
+This is a temporary table of contents, used during the development phase.
+When completed, the table of contents will be written in pydoc/index.rst
+and built with sphinx.
+
+Table of contents
+-----------------
+
+1. Introduction
+    * What is DLite and what is it used for?
+    * [Features](features.md)
```

Comment:
This be easier to read if we change it to [Features] and then add the following line at the end of the document:

[Features]: features.md

We want the links to point to the generated html-documentation, so the links at the end of the document must be updated when we know where the generated html files ends up on GitHub Pages.

Same comment for all other links

Reply:
It seems like the line to be added got lost in the rendering of your comment (?).

The html-documentation will be generated by sphinx, which has a special syntax for the table of contents (which enables correct links to html once sphinx has been run). So, don't you think it makes sense to keep the links to the markdown files at this stage?

Reply:
Keeping inline links to markdown files. This simplifies the creation of the sphinx version of the table of contents.